snappy/overlord.go

```diff
@@ -460,10 +460,10 @@ func (o *Overlord) InstallWithSideInfo(snapFilePath string, sideInfo *snap.SideI
 	}
 
 	// deal with the data
-	err = CopyData(newInfo, oldInfo, flags, meter)
+	err = copyData(newInfo, oldInfo, flags, meter)
```

Comment:
I wonder if we could simply remove `copyData` here (and remove Install/Uninstall along the way. The only user for this is ubuntu-device-flash but that is not able to create images with the latest snappy anyway because of the kernel.yaml changes, it tries to read meta/kernel.yaml from a mounted snap but in u-d-f we are not mounting so that will fail. I think we need a new special `bootstrap.Bootstrap()` for u-d-f anyway so we might remove these bits here now.